src/functionalTest/java/uk/gov/hmcts/reform/amapi/functional/AccessManangementApiTest.java

```diff
@@ -41,7 +41,7 @@ public void verifyGrantExplicitAccessApi() {
             + "}";
         try {
             Response response = amApiClient.createExplicitAccess().post(amApiClient.getAccessUrl()
-                + "api/access-resource");
+                + "api/v1/access-resource");
```

Comment:
Please define a global "version" variable and replace all of the version hardcoding in this file